appinventor/components/src/com/google/appinventor/components/runtime/Clock.java

```diff
@@ -605,6 +607,21 @@ public static String FormatTime(Calendar instant) {
     return Dates.FormatTime(instant);
   }
 
+  /**
+   * Changes the timezone of a given instant.
+   *
+   * @param instant : instant to modify
+   * @param timezone : timezone to apply to instant
+   * @return : updated instant
+   */
+  @SimpleFunction(description = "Updates the timezone in which the instant is saved at")
+  public static Calendar ChangeTimezone(Calendar instant, String timezone) {
+    if (Arrays.asList(TimeZone.getAvailableIDs()).contains(timezone))
```

Comment:
I'm wondering if it makes sense to cache this the first time this method is called so it doesn't have to be constructed every time someone calls ChangeTimezone?

Reply:
Got, I just changed it. This weekend I'll check if there's anything else I can do in the PR, and mark it as ready for review.